src/pages/PageLayout.tsx

```diff
@@ -1,39 +1,148 @@
-import { FC } from "react"
-import { Container, ContainerProps } from "@threshold-network/components"
-import { Outlet } from "react-router-dom"
-import SubNavigationPills from "../components/SubNavigationPills"
-import { PageComponent } from "../types"
-import useDocumentTitle from "../hooks/useDocumentTitle"
-
-export interface PageLayoutProps extends ContainerProps {
-  pages?: PageComponent[]
-  title?: string
+import {
+  Box,
+  Flex,
+  FlexProps,
+  Grid,
+  SystemStyleObject,
+  useMediaQuery,
+} from "@threshold-network/components"
+import { FC, ReactNode } from "react"
+import PRIMARY_BACKGROUND_PATH from "../static/images/layout-background-primary.svg"
+import SECONDARY_BACKGROUND_PATH from "../static/images/layout-background-secondary.svg"
+import { spacing } from "@chakra-ui/theme/foundations/spacing"
+import { Header } from "../components/Header"
+
+const CONTENT_MAX_WIDTH = "89.25rem" // 1428px
```

Comment:
Can you explain how did you come up with `CONTENT_MAX_WIDTH` value ("89.25rem" // 1428px) exactly? 🤔

Reply:
From Figma. Measured distance between most outer vertical borders.
<img width="1024" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/11503879/e096bccb-b57a-4a9e-a381-ba764341bde9">